.luacheckrc

```diff
@@ -0,0 +1,27 @@
+globals = {
+    "box",
+    "_TARANTOOL",
+}
+
+ignore = {
+    -- Unused argument <self>.
+    "212/self",
+    -- Redefining a local variable.
+    "411",
+    -- Shadowing a local variable.
+    "421",
+    -- Shadowing an upvalue.
+    "431",
+    -- Shadowing an upvalue argument.
+    "432",
```

Comment:
Let's without ignores at now.
```suggestion
```

Reply:
Left 432. It is used in server_helper from tarantool repo.